{"schemaVersion":{"minor":3,"patch":0,"major":0},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/recoverySuggestion-8rnsl"},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor","doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort","doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/DebuggableError-Implementations"]]},"sections":[],"metadata":{"title":"recoverySuggestion","externalID":"s:5Vapor15DebuggableErrorPAAE18recoverySuggestionSSSgvp::SYNTHESIZED::s:5Vapor5AbortV","extendedModule":"Vapor","role":"symbol","roleHeading":"Instance Property","modules":[{"name":"Vapor"}],"symbolKind":"property","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}]},"abstract":[{"text":"A localized message describing how one might recover from the failure.","type":"text"}],"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"platforms":["Linux"],"languages":["swift"]}],"kind":"declarations"}],"variants":[{"paths":["\/documentation\/vapor\/abort\/recoverysuggestion-8rnsl"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://co.pointfree.VaporRouting/documentation/Vapor/Abort/recoverySuggestion-8rnsl":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/recoverySuggestion-8rnsl","role":"symbol","kind":"symbol","title":"recoverySuggestion","url":"\/documentation\/vapor\/abort\/recoverysuggestion-8rnsl","abstract":[{"text":"A localized message describing how one might recover from the failure.","type":"text"}],"type":"topic","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}]},"doc://co.pointfree.VaporRouting/documentation/Vapor":{"type":"topic","abstract":[],"role":"collection","url":"\/documentation\/vapor","kind":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor","title":"Vapor"},"doc://co.pointfree.VaporRouting/documentation/Vapor/Abort/DebuggableError-Implementations":{"type":"topic","abstract":[],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort\/DebuggableError-Implementations","title":"DebuggableError Implementations","url":"\/documentation\/vapor\/abort\/debuggableerror-implementations","kind":"article","role":"collectionGroup"},"doc://co.pointfree.VaporRouting/documentation/Vapor/Abort":{"kind":"symbol","url":"\/documentation\/vapor\/abort","type":"topic","navigatorTitle":[{"text":"Abort","kind":"identifier"}],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/Vapor\/Abort","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Abort","kind":"identifier"}],"abstract":[{"type":"text","text":"Default implementation of "},{"type":"codeVoice","code":"AbortError"},{"type":"text","text":". You can use this as a convenient method for throwing"},{"type":"text","text":" "},{"type":"codeVoice","code":"AbortError"},{"type":"text","text":"s without having to conform your own error-type to "},{"type":"codeVoice","code":"AbortError"},{"type":"text","text":"."}],"title":"Abort","role":"symbol"}}}